designer/client/components/Fee/FeeItems.jsx

```diff
@@ -221,6 +264,66 @@ export class FeeItems extends React.Component {
                     ))}
                   </select>
                 </td>
+                <td className={"govuk-table__cell"}>
+                  {this.itemHasStaticQuantity(item) ? (
+                    <>
+                      <input
+                        className={classNames({
+                          "govuk-input": true,
+                          "govuk-input--error":
+                            errors?.[INVALID_MULTIPLIER]?.[index],
+                        })}
+                        name="multiplier"
+                        id="multiplier"
+                        type="number"
+                        defaultValue={item.multiplier ?? 0}
+                        step="any"
+                      />
+                      {fields.length > 0 && (
+                        <p className={"govuk-body govuk-!-margin-top-1"}>
+                          <a
+                            href={"#"}
+                            role="button"
+                            onClick={(e) =>
+                              this.changeQuantityType(e, item, index)
+                            }
+                          >
+                            Or choose a field
+                          </a>
+                        </p>
+                      )}
+                    </>
+                  ) : (
+                    <>
```

Comment:
I think if you're using fragments then it's worth just abstracting into its own component. that way you can manage errors individually as well (instead of checking indexes)?